apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -63,3 +80,35 @@ export const WarningLabel = ({children}) => (
 WarningLabel.propTypes = {
   children: React.PropTypes.node
 };
+
+/**
+ * Given a search query, generate a results list of animationProps objects that
+ * can be displayed and used to add an animation to the project.
+ * @param {string} searchQuery - text entered by the user to find an animation
+ * @return {Array.<SerializedAnimationProps>} - roughly
+ */
+function searchAnimations(searchQuery) {
+  // Make sure to generate the search regex in advance, only once.
+  // Search is case-insensitive
+  // Match any word boundary or underscore followed by the search query.
+  // Example: searchQuery "bar"
+  //   Will match: "barbell", "foo-bar", "foo_bar" or "foo bar"
+  //   Will not match: "foobar", "ubar"
```

Comment:
Wow!  That's an awesome idea, but maybe another pull request.  We're doing similar logging when we strip unexpected markup from applab design mode HTML to see what we might be mis-categorizing.  I'll take a look at this.